src/layouts/Titled/index.tsx

```diff
@@ -6,22 +6,29 @@
  */
 
 import React from 'react';
-import { css, ThemeProps, DefaultTheme } from 'styled-components';
-import { getColor, getLineHeight } from '@zendeskgarden/react-theming';
-import { LG } from '@zendeskgarden/react-typography';
+import styled, { css } from 'styled-components';
+import { getColor } from '@zendeskgarden/react-theming';
 import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { LG, XXXL } from '@zendeskgarden/react-typography';
 import { TOCBlock, TOC, IHeading } from './components/TOC';
 
-const headerStyles = (p: ThemeProps<DefaultTheme>) => {
-  const fontSize = `${p.theme.space.base * 12}px`;
+const TitleContainer = styled.div`
+  margin-bottom: ${p => p.theme.space.lg};
+  border-bottom: solid 1px ${p => getColor('grey', 300, p.theme)};
+  padding-bottom: ${p => p.theme.space.md};
+`;
 
-  return css`
-    margin-bottom: ${p.theme.space.md};
-    line-height: ${getLineHeight(`${p.theme.space.base * 14}px`, fontSize)};
-    font-size: ${fontSize};
-    font-weight: ${p.theme.fontWeights.bold};
-  `;
-};
+const PageTitle = styled(XXXL)`
+  margin-bottom: ${p => p.theme.space.sm};
+  line-height: 1;
+  font-size: ${p => p.theme.space.base * 12}px;
+  font-weight: ${p => p.theme.fontWeights.semibold};
```

Comment:
These custom `line-height` and `font-size` values now depart from the global typography sizing that matches the home page. Is this intended?